src/main/kotlin/nexters/admin/domain/member/Member.kt

```diff
@@ -0,0 +1,32 @@
+package nexters.admin.domain.member
+
+import javax.persistence.*
+
+@Entity
+@Table(name = "member")
+class Member(
+        @Column(name = "username", nullable = false)
+        val username: String,
+
+        @Column(name = "password", nullable = false)
+        var password: Password,
+
+        @Column(name = "email", nullable = false)
+        var email: String,
+
+        @Column(name = "gender", nullable = false, length = 10)
+        val gender: Gender,
```

Comment:
이게 DB에 MALE, FEMALE처럼 저장되는건가요 아니면 "남자", "여자"로 저장되는건가요? 

enum 쓸때 `@Enumerated` 어노테이션이 필요했던 것 같아서요.

```suggestion
        @Enumerated(EnumType.STRING)
        @Column(name = "gender", nullable = false, length = 10)
        val gender: Gender,
```

Reply:
큰일날 뻔 했네요. 
위와 같이 `@Enumerated`가 없으면 인덱스 숫자값이 저장됩니다.
바로 반영하겠습니다.